docs/README.md

```diff
@@ -12,6 +12,6 @@ Flarum is the combined successor of [esoTalk](https://github.com/esotalk/esoTalk
 
 * **Beautiful and responsive.** This is forum software for humans. Flarum is carefully designed to be consistent and intuitive across platforms, out-of-the-box.
 
-* **Powerful and extensible.** Customize, extend, and integrate Flarum to suit your community. Flarum’s architecture is amazingly flexible, with a [powerful Extension API](/extend/).
+* **Powerful and extensible.** Customize, extend, and integrate Flarum to suit your community. Flarum’s architecture is amazingly flexible, with a [powerful Extension API](https://github.com/flarum/docs/tree/master/docs/extend).
```

Comment:
What's the reasoning behind this change? I think it would be better for this to be an internal link to the extension docs rather than the code. Once we introduce official API docs (#122), we could link there as well

Reply:
Oh, I see. I clicked on the link here in the repo and thought it's broken (not thinking about the relative host on the site). How about we make it an absolute link to the docs? Then it works for both.

Reply:
I've tweaked it quickly to ensure it works on both @askvortsov1

Reply:
Sorry for the confusion (I'm jumping between a bunch of things today, and it's showing unfortunately :see_no_evil:). Wdym by "clicked on the link here in the repo"? Is there a use case for people acessing documentation links from within the github repo?

Reply:
Hey @askvortsov1, I know that feeling. I jump around too much as well.  by "clicked on the link here in the repo" here https://github.com/flarum/docs. It's the same page, in different context.

Reply:
I've tweaked it back to allow local editing over fixing the link on GitHub (makes sense).